src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs

```diff
@@ -54,52 +61,185 @@ public enum State
 
         #region Properties
 
+        public DynamoViewModel DynamoViewModel { get; private set; }
+
         private double zIndex;
+
+        public double ZIndex
+        {
+            get => zIndex;
+            set
+            {
+                zIndex = value;
+                RaisePropertyChanged(nameof(ZIndex));
+            }
+        }
+
         private Style infoBubbleStyle;
+
+        public Style InfoBubbleStyle
+        {
+            get { return infoBubbleStyle; }
+            set
+            {
+                infoBubbleStyle = value;
+                RaisePropertyChanged("InfoBubbleStyle");
```

Comment:
`nameof`

Reply:
This is another old one, the style itself never changes e.g. from Warning to Error. 

https://github.com/DynamoDS/Dynamo/blob/65b703d359401ca9646c65a96250c7085fd3d351/src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs